txpipe/diagnostic_maps.py

```diff
@@ -390,6 +394,28 @@ def copy_attrs(tomo, name, out_name):
 
         meta_file.close()
 
+        # human readable version
+        yaml_out_name = self.get_output('tracer_metadata_yml')
+        metadata = {
+            'lens_density': lens_density.tolist(),
+            'source_density': source_density.tolist(),
+            'sigma_e': shear_tomo_file['tomography/sigma_e'][:].tolist(),
+            'n_eff': n_eff.tolist(),
+            'R_gamma_mean': shear_tomo_file['metacal_response/R_gamma_mean'][:].tolist(),
+            'R_S': shear_tomo_file['metacal_response/R_S'][:].tolist(),
+            'R_total': shear_tomo_file['metacal_response/R_total'][:].tolist(),
+            'lens_counts': lens_counts.tolist(),
+            'source_counts': source_counts.tolist(),
+            'area': float(area),
```

Comment:
it's probably obvious, but do we want to store the units?

Reply:
added in branch redmagic_lens